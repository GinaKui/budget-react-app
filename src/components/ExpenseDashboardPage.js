import React, { Fragment } from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => (
  <main>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
  </main>
);

export default ExpenseDashboardPage;