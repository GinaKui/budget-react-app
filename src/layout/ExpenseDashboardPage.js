import React, { Fragment } from 'react';
import ExpenseList from '../components/ExpenseList';
import ExpenseListFilters from '../components/ExpenseListFilters';
import ExpensesSummary from '../components/ExpensesSummary';

const ExpenseDashboardPage = () => (
  <main>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
  </main>
);

export default ExpenseDashboardPage;