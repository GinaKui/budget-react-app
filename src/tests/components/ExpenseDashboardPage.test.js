import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../layout/ExpenseDashboardPage';

test('should render ExpenseDashboardPage', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});