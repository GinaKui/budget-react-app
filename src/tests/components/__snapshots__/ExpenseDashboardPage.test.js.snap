// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage 1`] = `
<main>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</main>
`;
